app/controllers/alaveteli_pro/stripe_webhooks_controller.rb

```diff
@@ -4,9 +4,26 @@ class AlaveteliPro::StripeWebhooksController < ApplicationController
 
   before_action :read_event_notification
 
+  class UnhandledStripeWebhookError < StandardError ; end
+
   def receive
     return unless @stripe_event
 
+    begin
+      case @stripe_event.type
```

Comment:
I'm able to get an exception here by not passing a `type` JSON field in the payload. Would be good to catch this.